src/util/branding.ts

```diff
@@ -122,6 +123,14 @@ function setBrandingPaletteColor(
   }
 }
 
+function setBrandingConfigPath(
+  brandingConfig: Record<string, unknown>,
+  configPath: string,
+) {
+  const path = buildPath(baseUrl.href, configPath);
+  brandingConfig["configPath"] = path.endsWith("/") ? path : path + "/";
```

Comment:
Better drop trailing `/` to make concatenations using a `/ ` as separator explicit.